activerecord/lib/active_record/locking/optimistic.rb

```diff
@@ -190,6 +190,11 @@ def deserialize(value)
         super.to_i
       end
 
+      def serialize(value)
+        # `nil` *should* be changed to 0
```

Comment:
A similar comment still exists on deserialize. Should that suffice, or it needs to be added here as well?